osu.Game/Storyboards/Commands/StoryboardTriggerGroup.cs

```diff
@@ -1,16 +1,16 @@
 ﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
-namespace osu.Game.Storyboards
+namespace osu.Game.Storyboards.Commands
 {
-    public class CommandTrigger : CommandTimelineGroup
+    public class StoryboardTriggerGroup : StoryboardCommandGroup
```

Comment:
What's with this rename? A "trigger group" does not seem like it could ever be the same thing as a "command trigger"...?

Reply:
a "trigger group" (people just call the triggers) is a group of commands that all activate when the trigger condition is met. the commands within a trigger are indented in the .osu/.osb files. it makes conceptual sense if anything

Reply:
As per the above comment, the term "trigger group" is better at describing how this class functions than "command trigger" and remains consistent with loop groups name-wise (`StoryboardLoopingGroup`).